### PR TITLE
INT-4467 - Add `external` property to `azure_user`

### PR DESCRIPTION
**Pull request by austinkelleher**
Example question + J1 query:

> Which Azure users are external?

```
FIND azure_user WITH external = true
```

### CODE REVIEW

File: src/steps/active-directory/converters.ts [hunk 1]
```diff
@@ -104,6 +104,10 @@ export function createGroupEntity(data: Group): Entity {
   });
 }
 
+function isExternalUser(username?: string) {
+  return !!(username && username.includes('#EXT#'));
```
Comment (austinkelleher) [inline src/steps/active-directory/converters.ts:108]: FYI I found an example in dev that includes `#EXT` in the username.

File: package.json [hunk 2]
```diff
@@ -87,12 +87,12 @@
...
-    "@jupiterone/integration-sdk-core": "^8.16.0"
+    "@jupiterone/integration-sdk-core": "^8.19.0"
   },
   "devDependencies": {
-    "@jupiterone/integration-sdk-core": "^8.16.0",
-    "@jupiterone/integration-sdk-dev-tools": "^8.16.0",
-    "@jupiterone/integration-sdk-testing": "^8.16.0",
+    "@jupiterone/integration-sdk-core": "^8.19.0",
+    "@jupiterone/integration-sdk-dev-tools": "^8.19.0",
+    "@jupiterone/integration-sdk-testing": "^8.19.0",
```
Comment (austinkelleher) [inline package.json:95]: Small bonus upgrades

File: src/steps/active-directory/converters.ts (hunk 1 above)
Comment (ndowmon) [reply src/steps/active-directory/converters.ts:108]: why not just do `return (data.userType != 'Member')`?

Comment (austinkelleher) [reply src/steps/active-directory/converters.ts:108]: Are you confident that these are the same thing? If so, happy to update.

Comment (ndowmon) [reply src/steps/active-directory/converters.ts:108]: Yep. Just invited an external user to my Azure tenant today 😉

Comment (austinkelleher) [reply src/steps/active-directory/converters.ts:108]: Fixed

**Comment by austinkelleher** [comment]
@zemberdotnet @VDubber @RonaldEAM @Nick-NCSU - Doing some open PR spring cleaning, but I still think this is a valuable improvement. 😬 🌷

**Pull request closed without merging**